cmd/brew-report-issue.rb

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env ruby
+# Creates and closes issues on a project.
+close = !!ARGV.delete("--close")
+user_repo = ARGV.shift
+failure_source = ARGV.shift
+success_meta = failure_source
+
+if user_repo.to_s.empty? || failure_source.to_s.empty?
+  abort "Usage: brew report-issue [--close] <user/repo> <failure-source|success-meta> [<STDIN piped body>]"
```

Comment:
I'm not sure `success-meta` is from reading this usage. Reading the code and being familiar with boxen, it's the sha it succeeded at?